Description: allow nm-applet to be displayed
 on Ubuntu nm-applet defaults to only be show by appindicators.
 For budgie-desktop, this needs to be supplied by the X11 system tray since
 appindicators are not a required dependency.
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: not-needed
Origin: Other
===================================================================
--- a/x/budgie-desktop-10.2.9/src/session/budgie-desktop-nm-applet.desktop.in
+++ b/x/budgie-desktop-10.2.9/src/session/budgie-desktop-nm-applet.desktop.in
@@ -2,7 +2,7 @@
 Name=Network (Budgie Desktop)
 Comment=Manage your network connections
 Icon=nm-device-wireless
-Exec=nm-applet
+Exec=nm-applet --no-indicator
 TryExec=nm-applet
 Terminal=false
 Type=Application
